src/App.php

```diff
@@ -126,6 +126,51 @@ class App
 	 */
 	private $pConfig;
 
+	/**
+	 * Set the user ID
+	 *
+	 * @param int $user_id
+	 * @return void
+	 */
+	public function setUserId(int $user_id)
+	{
+		$this->user_id = $user_id;
+	}
+
+	/**
+	 * Set the nickname
+	 *
+	 * @param int $user_id
+	 * @return void
+	 */
+	public function setNickname(string $nickname)
+	{
+		$this->nickname = $nickname;
+	}
+
+	public function isLoggedIn()
+	{
+		return local_user() && $this->user_id && ($this->user_id == local_user());
+	}
+
+	/**
+	 * Fetch the user id
+	 * @return int 
+	 */
+	public function getUserId()
```

Comment:
Is it the User id of the App? If not, please make the name more specific for its intended use. Doesn't it overlap with `local_user()` if it's the logged in user id?

Reply:
It totally overlaps. And I think that in the future we should use that function. That's one of the reasons why I wanted to keep the function name brief.

Reply:
I understand but it doesn't mean much in the current state. I would prefer something like `App->getLoggedInUserId()`

Reply:
I wouldn't want to do the `local_user()` change now. I'm sure that there will be side effects since the one value is fetched via a session and the other via the application class. Performing the change some days in the future is much better for debugging.

Reply:
```suggestion
	public function getLoggedInUserId()
```

Reply:
Done